ui-v2/tests/acceptance/dc/services/instances/show.feature

```diff
@@ -0,0 +1,68 @@
+@setupApplicationTest
+Feature: dc / services / instances / show: Show Service Instance
+  Scenario: A Service instance has no Proxy
+    Given 1 datacenter model with the value "dc1"
+    And 1 service model from yaml
+    ---
+    - Service:
+        ID: service-0-with-id
+        Tags: ['Tag1', 'Tag2']
+        Meta:
+          external-source: nomad
+      Checks:
+        - Name: Service check
+          ServiceID: service-0
+          Output: Output of check
+          Status: passing
+        - Name: Service check
+          ServiceID: service-0
+          Output: Output of check
+          Status: warning
+        - Name: Service check
+          ServiceID: service-0
+          Output: Output of check
+          Status: critical
+        - Name: Node check
+          ServiceID: ""
+          Output: Output of check
+          Status: passing
+        - Name: Node check
+          ServiceID: ""
+          Output: Output of check
+          Status: warning
+        - Name: Node check
+          ServiceID: ""
+          Output: Output of check
+          Status: critical
+    ---
+    And 1 proxy model from yaml
+    ---
+    - ServiceProxy:
+        DestinationServiceName: service-1
+        DestinationServiceID: ~
```

Comment:
Does this `~` do something magic (just curious)?

Reply:
Yeah so in YAML `~` means `null`, you can't use `null` as in YAML thats the same as `"null"`, some info here:

https://yaml.org/type/null.html

Specifically in the test, the way the mocks work is that they give you a full mocked out 'Consul object' and you can just overwrite the specific bits you need different for your test. This is handy as it means you don't have to write out all the YAML for the entire object you are testing - only the specific properties that you are testing.

That leads to you occasionally needing to 'unset' a property on an object, so null it out, so you use `~`